docs/en/observability/apm/manage-storage/ilm-how-to.asciidoc

```diff
@@ -18,18 +18,15 @@ A default policy is applied to each APM data stream, but can be customized depen
 [NOTE]
 ====
 // Explain what changed
-// Before
-Before 8.15, clusters used {ref}/index-lifecycle-management.html[index lifecycle management (ILM)] to provide the default data retention settings for APM data.
-// After
-As of 8.15, new clusters use {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] to provide default data retention settings, but the customization for lifecycle is still performed using ILM policies, which are now configured to override the default DSL polices.
+Clusters are managed by {ref}/index-lifecycle-management.html[index lifecycle management (ILM)] to provide the default data retention settings for APM data as well as allow customization for lifecycle.
+
+New clusters created in 8.15 and 8.16 will use {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] to provide default data retention settings, but the customization for lifecycle is still performed using ILM policies, which are configured to override the default DSL polices. When these clusters are upgraded to 8.17 the default will revert back to ILM, however, the indices created will continue to use DSL. The changes only affect default lifecycle policies, so, if you have custom ILM policies configured then there are expected to be no changes and the custom ILM policy will continue to be used.
```

Comment:
I am having a hard time digesting this, so I assume our users would have to. Can we just remove any mentioning of DSL from here and only leverage release notes and known issues sections for references to DSL. 
As soon as they upgrade, we expect any datastreams to be managed by ILM again.

Reply:
Simplified to drop the entire notes section.